### PR TITLE
Android push notification missing native functions

### PR DESCRIPTION
**Pull request by mevans**
Pr to allow the calling of 2 native functions required for push notifications on Android:
 - `sendTokenToIntercom`
 - `handlePush`

This is android specific functionality isn't included in this plugin.

Takes heavy inspiration from https://github.com/capacitor-community/intercom/pull/41

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # node files
-dist
```
Comment (eljass) [inline .gitignore:2]: When merging to this repository it is not needed to commit dist folder as this is used via npm package and it does build its own dist when installing it. In our branch, we fetch the plugin straight from Github, and then there is a needed build step before committing to GH. 😄

**Comment by gcorreaalves** [comment]
Hello guys, any predictions of accepting this PR?

**Comment by AidenTate** [comment]
I'd also love for this to be merged please

**Comment by stewones** [comment]
please don't add `dist` folder to git. I'll merge and remove from master

**Pull request merged**